spec/components/candidate_interface/continuous_applications/course_review_component_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::ContinuousApplications::CourseReviewComponent do
+  describe 'course_fee' do
+    let(:application_choice) do
+      create(:application_choice,
+             current_course_option: create(:course_option, course: create(:course, fee_domestic:, fee_international:)))
+    end
+
+    context 'where domestic and international fees present' do
+      let(:fee_domestic) { 9250 }
+      let(:fee_international) { 23820 }
+
+      it 'shows both domestic and international fees' do
+        result = render_inline(described_class.new(application_choice:))
+        expect(result.text).to include 'UK students: £9,250'
+        expect(result.text).to include 'International students: £23,820'
+      end
+    end
+
+    context 'where only domestic fees are present' do
+      let(:fee_domestic) { 9250 }
+      let(:fee_international) { nil }
+
+      it 'shows both domestic and international fees' do
```

Comment:
Can we update this spec description to something like `it only shows domestic fees` same on line 36.

Reply:
resolved with https://github.com/DFE-Digital/apply-for-teacher-training/pull/9153/commits/25e70ff6dd0ee4873bbd2c4f4bc5a6379aa393aa